pkg/commandRunner/cmdrunner.go

```diff
@@ -34,11 +34,11 @@ func (c *CommandRunner) ExecuteAndLog(dir, name string, args ...string) error {
 	log.Logger().Infof("%sRunning command%s:%s %s %s", utils.Cyan, dirString, utils.Reset, name, strings.Join(args, " "))
 	out, err := c.Execute(dir, name, args...)
 	if err != nil {
-		log.Logger().Errorf(out)
+		log.Logger().Errorf("%s", out)
```

Comment:
And this

Reply:
ye went through them thought there was others changed the others but must of missed this one , ahhh